src/loader.js

```diff
@@ -43,28 +43,30 @@ export class Loader {
       BufferLayout.u32('bytesLengthPadding'),
       BufferLayout.seq(
         BufferLayout.u8('byte'),
-        BufferLayout.offset(BufferLayout.u32(), -8),
-        'bytes'
-      ),
+        BufferLayout.offset(BufferLayout.u32(), -8),'bytes'),
     ]);
 
-    let userdata = Buffer.alloc(bytes.length + 16);
-    userdataLayout.encode(
-      {
-        instruction: 0, // Load instruction
-        offset,
-        bytes,
-      },
-      userdata,
-    );
+    let i,j;
+    const chunk_size = 256; // Size of chunk, just needs to fit into tx
+    let userdata = Buffer.alloc(chunk_size + 16);
+    for (i = 0, j = bytes.length; i < j; i += chunk_size) {
```

Comment:
I think we can replace ` for (i = 0, j = bytes.length; i < j; i += chunk_size) {` with

```js
const offset = 0;
while (bytes.length > 0) {
  const slice = bytes.slice(0, chunkSize);

  // send `slice`

  offset += chunkSize;
  bytes = bytes.slice(chunkSize);
}
```